src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java

```diff
@@ -93,4 +93,20 @@ CaseListResponse getCaseList(@PathVariable String courtCode,
                 .collect(Collectors.toList());
         return CaseListResponse.builder().cases(courtCaseResponses).build();
     }
+
+    @ApiOperation(value = "For the date / cases passed, delete any cases NOT in the list.")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 204, message = "OK", response = CaseListResponse.class),
```

Comment:
Response class is incorrect, think we can remove this line as it's derived automatically anyway and very easy to get wrong!

Reply:
Done